docker-compose.yml

```diff
@@ -22,6 +21,8 @@ services:
       DB_DATABASE: postgres
       DB_HOST: postgres
       DB_PORT: 5432
+    volumes:
+      - ./:/app
```

Comment:
do we really need these changes?

Reply:
No, we don't,
had issues during the testing => switched to the default `docker-compose.yml`